### DISCUSSION TITLE
Debug main process in vscode

### DISCUSSION THREAD

**Discussion by lacoonte** [cawa-93/vite-electron-builder#541] [Q&A] [answered]
I have been trying to attach my debugger with no results. If you have some launch.json/task.json compatible with this template it would be fantastic. Thanks in advance!

**Comment by vdawg-git** [comment]
I do not have an answer but a helpful tip: [`Ololog`](https://github.com/xpl/ololog) is a logger which automatically detectes source-maps and logs its output nicely formatted with the original filename and line number.
It is not as good as having a debugger, but helps a lot :D

**Comment by cawa-93** [comment, marked as answer]
Should be resolved by https://github.com/cawa-93/vite-electron-builder/commit/51236a693761a56fea05655abc0f6f1632074501
